SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -75,48 +104,82 @@ They work for:
 These macros replace %%py3_build and %%py3_install,
 which only work with setup.py.
 
+
+%package -n pyproject-srpm-macros
+Summary:        Minimal implementation of %%pyproject_buildrequires
+Requires:       pyproject-rpm-macros = %{?epoch:%{epoch}:}%{version}-%{release}
+Requires:       rpm-build >= 4.14.90
+
+%description -n pyproject-srpm-macros
+This package contains a minimal implementation of %%pyproject_buildrequires.
+When used in %%generate_buildrequires, it will generate BuildRequires
+for pyproject-rpm-macros. When both packages are installed, the full version
+takes precedence.
+
+
 %prep
 # Not strictly necessary but allows working on file names instead
 # of source numbers in install section
-%setup -q -c -T
+%setup -c -T
 cp -p %{sources} .
 
+%generate_buildrequires
+# nothing to do, this is here just to assert we have that functionality
+
```

Comment:
I think it would be better to remove these lines, since the comment is misleading. We (Azure Linux) actually do not support `generate_buildrequires` right now. As in: RPM will work with it correctly but it's ignored by out toolkit's package build.

Reply:
removed